### PR TITLE
News(idea) #80 resolved

### PR DESCRIPTION
**Pull request by Rits1272**
Closes: #80 

## Proposed changes

This script gives news in the target whatsapp group. Currently the news is basically live corona virus news. The user needs to specify country code. After that the script gives live updates of corona virus on that particular country after every 15 minutes

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
WhatsApp Image 2020-03-27 at 4 33 36 AM

WhatsApp Image 2020-03-27 at 4 33 36 AM (1)

## Use
Run `python3 -m wplay -wnews "Anyname"`

### CODE REVIEW

File: wplay/wnews.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    page, _ = await browser_config.configure_browser_and_load_whatsapp()
+    if target is not None:
+        try:
+            await target_search.search_and_select_target(page, target)
+        except Exception as e:
+            print(e)
+            await target_search.search_and_select_target_without_new_chat_button(page, target)
+    else:
+        await target_select.manual_select_target(page)
+
```
Comment (rpotter12) [inline wplay/wnews.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5254601)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- wplay/wnews.py  5
- wplay/__main__.py  1
         

Clones added
============
- wplay/wnews.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5254601&bid=17217520)

File: wplay/wnews.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        time.sleep(900) # Sends news in every 15 min
+
+
+def fetch_news(country_code):
+    try:
+        data = requests.get(f"https://thevirustracker.com/free-api?countryNewsTotal={country_code}", headers={"User-Agent": "XY"}).json()
+        news = data['countrynewsitems'][0]['1']['title']
+        source = data['countrynewsitems'][0]['1']['url']
+        return news, source
+    except:
```
Comment (rpotter12) [inline wplay/wnews.py]: Codacy Issue found: [No exception type(s) specified](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5254601)

**Comment by xandao6** [comment]
> It will show the news of virus only. Open a new issue to change it fetch the news of all types.

Yes, will be nice, you can also do this change in the status changer @Rits1272

**Pull request merged**